doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -1,5 +1,3 @@
-:orphan:
-
 .. _ug_nrf54h20_gs:
 
 Getting started with the nRF54H20 DK
```

Comment:
Link to here from https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/gsg_guides.html